app/Console/Commands/Temp/ResetPackageAvailabilityStatus.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Console\Commands\Temp;
+
+use App\Package;
+use Illuminate\Console\Command;
+
+class ResetPackageAvailabilityStatus extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'novapackages:reset-package-availability';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Reset package availability status';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $packages = Package::withoutGlobalScopes()->get();
```

Comment:
In the future, just FYI, this command could be:

```php
        Package::withoutGlobalScopes()->update([
            'marked_as_unavailable_at' => null,
        ]);
```

This doesn't trigger a `save` event and doesn't get you your `info`, but it's faster and simpler :)